README.md

```diff
@@ -33,6 +33,9 @@ In case you have already checked out the repo, run the following command:
 pod install --project-directory=catalog/
 ~~~
 
+If you want to take a look at the implementation of the components, you can find the code inside the development pod.
+Scroll down the left xCode panel after running the above command to find the code.
```

Comment:
`cmd-1` to open the project navigator within Xcode. Peal open the `Pods` project and inside the `Development Pods` folder you will find the component source code.